reviews/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db              import models
+from django.core.validators import MinValueValidator, MaxValueValidator
+
+from core.models import TimeStampModel
+
+class Review(TimeStampModel):
+    content          = models.TextField()
+    accuracy         = models.IntegerField(validators=[MinValueValidator(1), MaxValueValidator(5)])
+    communication    = models.IntegerField(validators=[MinValueValidator(1), MaxValueValidator(5)])
+    cleanliness      = models.IntegerField(validators=[MinValueValidator(1), MaxValueValidator(5)])
+    location         = models.IntegerField(validators=[MinValueValidator(1), MaxValueValidator(5)])
+    check_in         = models.IntegerField(validators=[MinValueValidator(1), MaxValueValidator(5)])
+    cost_performance = models.IntegerField(validators=[MinValueValidator(1), MaxValueValidator(5)])
+    user             = models.ForeignKey('users.User', related_name='reviews', on_delete=models.CASCADE)
+    room             = models.ForeignKey('rooms.Room', related_name='reviews', on_delete=models.CASCADE)
+    
+    class Meta:
+        db_table = 'reviews'
+
+class ReviewImages(models.Model):
+    image_url = models.CharField(max_length=200)
+    review    = models.ForeignKey('Review',related_name='reviewimages', on_delete=models.CASCADE)     
```

Comment:
```suggestion
    review    = models.ForeignKey('Review',related_name='review_images', on_delete=models.CASCADE)     
```
related_name 지을 때도 단어 사이에서는 _을 붙여주세요.(이 부분 말고도 있으면 다 수정해 주세요.)

Reply:
넵 알겠습니다~~!!!